### ISSUE TITLE
BHB10K dataset for pretraining.

### ISSUE THREAD

**Issue by aneesh-aparajit** [Duplums/yAwareContrastiveLearning#1]
I am currently working on contrastive learning with SimCLR on the Alzheimer's data set. I have tried the pre-trained weights, but the validation loss doesn't seem to converge quite well. Would it be possible to access to the BHB-10K data, for further pre-training?

**Comment by Duplums** [comment]
Thanks for your interest ! Data from BHB-10K cannot necessarily be shared but you can find an opened version here: https://ieee-dataport.org/open-access/openbhb-multi-site-brain-mri-dataset-age-prediction-and-debiasing 
I hope this could help.

**Comment by aneesh-aparajit** [comment]
Thanks a lot!

**Comment by aneesh-aparajit** [comment]
I also noticed that your code also fine tunes the encoder and not just the classifier. Which I believe is not the conventional way of doing contrastive learning. Is there any specific reason for doing so?

**Comment by Duplums** [comment]
This code allows you to fine-tune the whole encoder to perform Transfer Learning on a downstream task (it is standard to fine-tune all weights in that case). This is conventional for practioners working on specific pathologies (e.g. Alzheimer's disease) as it usually gives better results than simple linear evaluation (where only a linear layer is trained on top of the pre-trained encoder). The code for linear evaluation can be found in several standard repos on CL (e.g [SimCLR]https://github.com/sthalles/SimCLR) ).

**Comment by aneesh-aparajit** [comment]
Oh thanks! I was unaware of this convention. Pretty new to the field. Thanks a lot! Really Appreciate it! Thanks!
